src/MigrationLog/MigrationLogModel.php

```diff
@@ -29,6 +29,11 @@ class MigrationLogModel {
 	 */
 	private $error;
 
+	/**
+	 * @var int
+	 */
+	private $run_order;
+
```

Comment:
Is this still used?

Reply:
No, that one was left behind. I'll remove that.

Reply:
Actually, it's already removed 
14a4c7e

Reply:
Looking at that commit, it looks like you removed it's usage but not the property definition.

Reply:
Now its removed for real 7e2013c3